GOVERNANCE.md

```diff
@@ -25,6 +25,15 @@ StackStorm uses a three-tiered system of Maintainer roles:
   * Have good experience with the StackStorm codebase, expected to provide significant value to the project, helping it grow, improve and succeed.
   * Have full member write access to [StackStorm](https://github.com/stackstorm/) and [StackStorm-Exchange](https://github.com/stackstorm-exchange) Github organizations, CI/CD, Moderator at [forum](https://forum.stackstorm.com/), [Slack](https://stackstorm.com/community-signup) and other Community platforms.
   * Receive **one vote** in the voting process.
+* [Chair](OWNERS.md#chair-)
```

Comment:
@armab I don't think the spirit of this role is actually to replace the `Leaders` role. My understanding is it is a role for someone to volunteer and be elected to run the TSC meetings on behalf of the `TSC group as a whole`. While some of the Leaders responsibilities may be delegated to this role, it's not my understanding that it would replace Dmitri as Leader of the project. cc @StackStorm/tsc 

Maybe the the role of `Secretary` would better communicate this.

Reply:
Does this mean that the chair gets the 3 votes that were normally given to the Leadership role?

Reply:
I originally though we still had the leadership role, but the chair was a responsibility that we'd alter every n months. Keeping the leadership role would keep continuity.
So I like @punkrokk difference between position/role, but prefer the position to be named chair than secretary, as I think it is more than what you'd associate with a secretary position.

Reply:
I've attended KubeCon/CloudNativeCon and 2 OpenSource Summits this year which are all about the LF and CNCF projects 

I would say that TSC Chair is pretty much a Leadership role for the project, apart of just running the meetings.
It's literally head of the TSC. During the nomination phase TSC Chair normally provides his "statement of intent" describing the project vision, plans, etc. And so why Election is important.

Now look at the [Maintainer Roles](https://github.com/StackStorm/st2/blob/master/GOVERNANCE.md#maintainer-roles):
```
Leaders
    Head of Technical Steering Committee (TSC).
```

When Governance was drafted, initially we decided instead of adding 1 TSC Chair to add many Leaders that would be responsible for all this. We defined _many_ Leader(s) to make sure it fits @dzimine and @LindsayHill. Now if we add one TSC Chair, it naturally should replace the Leaders per original idea and consistency.

So or define _one_ TSC Chair or keep _many_ TSC Leaders as is.

Reply:
Here are more questions:

- If we want to keep the `Leaders`, can the `TSC Chair` be also part of the `Leaders` group?
  - If not, why?
  - If yes, isn't it confusing for 1 person to be part of 2 groups/roles/positions/whatever?
- If TSC Chair is an overall a Leadership role, is he automatically a member of the Leaders too?
  - If not, why?
  - If yes, why another chair role needed?
- If we add the TSC Chair just as a meeting runner (which I believe is not what Project Chairperson is), why this couldn't be distributed between the TSC Leaders in a more simple manner?


What the existing Governance tells:
* **Leaders**
    * [...]
    * Responsible for Project Strategy, External Relations, _Organizational aspects_, _coordinating Events_, Partnerships.

Based on that, Isn't a TSC Meeting an _Organizational aspect_ and _Coordination Event_?
TSC Chair is normally responsible for External Relations as well. This is what LF recommends. Do you see now these signs that Leaders were created to reflect the TSC chairperson responsibilities? 

^^ Having said that, to keep the Governance consistent, this needs more work around re-defining roles. Otherwise we'll have more confusion rather than clarity with 5 different roles.